plugins/intel-spi/generate-quirk.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/python3
+# pylint: disable=invalid-name,missing-docstring
+#
+# Copyright (C) 2021 Richard Hughes <richard@hughsie.com>
+#
+# SPDX-License-Identifier: LGPL-2.1+
+
+import sys
+
+
+class Chipset:
+    def __init__(self, spibar=None, bios_cntl=0x0, spibar_proxy=None, flags=None):
+
+        self.bios_cntl = bios_cntl
+        self.spibar_proxy = spibar_proxy
+        self.spibar = spibar
+        self.flags = flags
+
+
+if __name__ == "__main__":
+
+    if len(sys.argv) != 2:
+        print("required: /path/to/chipset_enable.c")
```

Comment:
this is kernel source right?  Just a thought - what about a daily or weekly CI job that is checking it against a file at https://github.com/torvalds/linux/ (or similar) and making noise when something is added?  Or even automatically _committing_ when something is added?

Reply:
Nah, this is from flashrom. I'm not 100% in agreement with the auto-sync yet, I'm pondering using flashrom *and* chipsec and take the superset of hardware.